src/lake/Lake/Load/Package.lean

```diff
@@ -107,6 +107,10 @@ def Package.finalize (self : Package) (deps : Array Package) : LogIO Package :=
         error s!"post-update hook was defined in `{decl.pkg}`, but was registered in `{self.name}`"
     | .error e => error e
 
+  -- Deprecation warnings
+  unless self.config.moreServerArgs.isEmpty do
+    logWarning s!"{self.name}: package configuration option `moreServerArgs` is deprecated in favor of `moreGlobalServerArgs`"
```

Comment:
This should suggest `moreServerOptions`, not `moreGlobalServerArgs`, since most uses of this flag in the wild as global server args are incorrect and following the suggestion will cause the same issue that this PR is trying to solve.